precompiles/staking/utils_test.go

```diff
@@ -405,22 +265,22 @@ func (s *PrecompileTestSuite) assertValidatorsResponse(validators []staking.Vali
 	// returning order can change
 	valOrder := []int{0, 1}
 	varAddr := sdk.ValAddress(common.HexToAddress(validators[0].OperatorAddress).Bytes()).String()
-	if varAddr != s.validators[0].OperatorAddress {
+	if varAddr != s.network.GetValidators()[0].OperatorAddress {
 		valOrder = []int{1, 0}
 	}
 	for i := 0; i < expLen; i++ {
 		j := valOrder[i]
 
-		s.Require().Equal(s.validators[j].OperatorAddress, sdk.ValAddress(common.HexToAddress(validators[i].OperatorAddress).Bytes()).String())
-		s.Require().Equal(uint8(s.validators[j].Status), validators[i].Status)
-		s.Require().Equal(s.validators[j].Tokens.Uint64(), validators[i].Tokens.Uint64())
-		s.Require().Equal(s.validators[j].DelegatorShares.BigInt(), validators[i].DelegatorShares)
-		s.Require().Equal(s.validators[j].Jailed, validators[i].Jailed)
-		s.Require().Equal(s.validators[j].UnbondingHeight, validators[i].UnbondingHeight)
+		s.Require().Equal(s.network.GetValidators()[j].OperatorAddress, sdk.ValAddress(common.HexToAddress(validators[i].OperatorAddress).Bytes()).String())
+		s.Require().Equal(uint8(s.network.GetValidators()[j].Status), validators[i].Status)
+		s.Require().Equal(s.network.GetValidators()[j].Tokens.Uint64(), validators[i].Tokens.Uint64())
+		s.Require().Equal(s.network.GetValidators()[j].DelegatorShares.BigInt(), validators[i].DelegatorShares)
+		s.Require().Equal(s.network.GetValidators()[j].Jailed, validators[i].Jailed)
+		s.Require().Equal(s.network.GetValidators()[j].UnbondingHeight, validators[i].UnbondingHeight)
```

Comment:
better to get the validator once and then reuse it


```suggestion
        val := s.network.GetValidators()[j]
		s.Require().Equal(val.OperatorAddress, sdk.ValAddress(common.HexToAddress(validators[i].OperatorAddress).Bytes()).String())
		s.Require().Equal(uint8(val.Status), validators[i].Status)
		s.Require().Equal(val.Tokens.Uint64(), validators[i].Tokens.Uint64())
		s.Require().Equal(val.DelegatorShares.BigInt(), validators[i].DelegatorShares)
		s.Require().Equal(val.Jailed, validators[i].Jailed)
		s.Require().Equal(val.UnbondingHeight, validators[i].UnbondingHeight)
```